packages/perseus/src/__tests__/validation.typetest.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * This file contains TypeScript type "tests" which ensure that types needed
+ * for scoring and validation stay in sync with other types in the system.
+ *
+ * If you make a change and one of these "satisfies" assertions fails, that
```

Comment:
May want to update this line since it no longer uses `satisifies` assertions, though I suppose you could still think of them that way.